packages/fx-core/src/component/driver/deploy/azure/impl/baseDeployImpl.ts

```diff
@@ -83,15 +87,15 @@ export abstract class BaseDeployImpl {
    */
   protected async packageToZip(args: DeployStepArgs, context: DeployContext): Promise<Buffer> {
     const ig = await this.handleIgnore(args, context);
-    const zipFilePath = path.join(
+    this.tmpName = path.join(
```

Comment:
zipFilePath looks more straightforward, what's the purpose of this.tmpName?

Reply:
Changed back to zipFilePath and moved this property and functions to ZipDeployImplement.